pkg/tsdb/mysql/mysql.go

```diff
@@ -32,6 +32,17 @@ func newMysqlQueryEndpoint(datasource *models.DataSource) (tsdb.TsdbQueryEndpoin
 		datasource.Url,
 		datasource.Database,
 	)
+
+	tlsConfig, err := datasource.GetTLSConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	if tlsConfig.RootCAs != nil || len(tlsConfig.Certificates) > 0 {
```

Comment:
This feels kind of dirty, but I don't want to do a `mysql.RegisterTLSConfig` with an empty tls config/tls config with only InsecureSkipVerify set. And I didn't find a nicer way of abstracting this.